index.html

```diff
@@ -15,31 +15,294 @@
-                <li class="icon"> <a href=""><i class="fa-solid fa-m"></i></a></li>
-                <li class="icon"> <a href="#https://github.com/TiteLunda"><i class="fa-brands fa-github"></i></a></li>
-                <li class="icon"> <a href=""><i class="fa-brands fa-angellist"></i></a></li>
-            </ul>
+            <nav class="menu">
+                <i class="fa-solid fa-bars "></i>
+            </nav>
+        </header>
+
+        <section class="header">
+            <div class="container">
+                <h1 class="identity">
+                    Hey there. I'm, <br/> Tite Lunda <br><span>I'm a software developer</span>
+                </h1>
+                <p class="profile">
+                    <b></b> I can help you build a product , feature or website Look through some of my work and experience! If you like <br/>what you see and have a project you need coded, don’t hestiate to contact me. </p>
+
+                <ul class="icon-group">
+                    <li class="icon"> <a href=""><i class="fa-brands fa-twitter"></i></a></li>
+                    <li class="icon"> <a href=""><i class="fa-brands fa-linkedin"></i></a></li>
+                    <li class="icon"> <a href=""><i class="fa-solid fa-m"></i></a></li>
+                    <li class="icon"> <a href="#https://github.com/TiteLunda"><i class="fa-brands fa-github"></i></a></li>
+                    <li class="icon"> <a href=""><i class="fa-brands fa-angellist"></i></a></li>
+                </ul>
+                <ul class="icon-group">
+                    <li><i class="fa-solid fa-chevron-down">
+                    </i>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="works">
+            <div class="Recent">
+                <h1>My Recents Works</h1>
+            </div>
+            <div class="cards">
+                <img src="/Images/Img Plaholder.png" alt="Store image">
+                <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                    <div class="Languages">
+                        <div class="language">Ruby on rails</div>
+                        <div class="language">Css</div>
+                        <div class="language">JavaScript</div>
+                        <div class="language">Html</div>
+                    </div>
+                    <button class="Read-more">See Project</button>
+            </div>
+
+            <div class="cards">
+                <img src="/Images/Img Plaholder.png" alt="Store image">
+                <h4 class="title">Multi-Post Stories Gain+Glory</h1>
+                    <div class="Languages">
+                        <div class="language">Ruby on rails</div>
+                        <div class="language">Css</div>
+                        <div class="language">JavaScript</div>
+                        <div class="language">Html</div>
+                    </div>
+                    <button class="Read-more">See Project</button>
```

Comment:
- [ ] I kindly suggest that you make your "See Project" button is aligned to the center as it is on the Figma design
- [ ] I kindly suggest that you make the image height similar to the Figma design
- [ ] I kindly suggest that you increase the font weight of the button like tag to match the Figma design
- [ ] Please kindly change the color of your project text to#3A4A42; to match the Figma design


| Your Project | Figma Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/52934128/219884720-ad1f6ada-4f02-449b-bdf0-3aad84e524f6.png)  | ![image](https://user-images.githubusercontent.com/52934128/219884688-9e293c1d-7a60-4ad7-8c95-797f24069749.png)  |